app/presenters/licences/purposes.presenter.js

```diff
@@ -11,16 +11,13 @@ const { formatAbstractionAmounts, pluralise } = require('./base-licences.present
 /**
  * Formats the licence and related licenceVersionPurposes data for the view licence purposes page
  *
- * @param {object} licence - The licence and related licenceVersionPurposes data returned by
- * `FetchLicencePurposesService`
+ * @param {object} licenceVersionPurposes - The licenceVersionPurposes data returned by `FetchLicencePurposesService`
```

Comment:
If you are going to be consistent, then this should be `purposes`.

Otherwise, the other services need to name their args `licenceVersionConditions` and `licenceVersionPoints`.